src/python/grpcio/grpc/_cython/_cygrpc/credentials.pyx.pxi

```diff
@@ -126,28 +126,69 @@ cdef class SSLSessionCacheLRU:
     grpc_shutdown()
 
 
+cdef int verify_peer_callback_wrapper(const char* servername, const char* cert, void* userdata) with gil:
+  if userdata == NULL:
+    print("Error! Callback function wasn't set!")
```

Comment:
`print`? From the Cython code? Rather than a logging statement or raising an exception?

Reply:
Oof, I should be embarrassed. Changed that to raising an exception.